package.json

```
@@ -35,12 +35,19 @@
 	},
 	"engines": {
 		"node": "^22.9.0",
-		"pnpm": "^9.15.0"
+		"pnpm": "^10.4.0"
 	},
-	"packageManager": "pnpm@9.15.0",
+	"packageManager": "pnpm@10.4.0",
 	"pnpm": {
 		"patchedDependencies": {
 			"@wordpress/dataviews": ".pnpm-patches/@wordpress__dataviews@4.12.0.patch"
-		}
+		},
+		"onlyBuiltDependencies": [
```

Since we don't seem to actually need any of these, I might do
```suggestion
		"ignoredBuildDependencies": [
```
instead. At the least we might drop `core-js`, `svlte-preprocess`, and `swiper` in there since their builds do nothing useful.

---

Yeah, I was hoping to do this (both with `ignoredBuiltDependencies` and `neverBuiltDependencies`), but when I ran it locally I was still getting the `ERR_PNPM_IGNORED_BUILDS` error, and I didn't see any support for overrides in a brief skim of the source (pnpm/pnpm#9071). Does it work for you? 🤔

---

Yeah, seems to work for me with `ignoredBuiltDependencies` and 10.4.0 or 10.4.1.

I had a typo earlier, copied from a typo in https://github.com/pnpm/pnpm/releases/tag/v10.4.1 😀

---

Looks like it was my end (surprise!); clearing `node_modules` fixed it.